tests/test-extensions/test-oauth-extension/src/main/java/org/mule/test/oauth/TestWithOAuthParamsConnectionProvider.java

```diff
@@ -39,6 +41,10 @@ public class TestWithOAuthParamsConnectionProvider extends LegacyTestOAuthConnec
   @OAuthParameter(placement = HttpParameterPlacement.HEADERS)
   String header;
 
+  @org.mule.sdk.api.annotation.connectivity.oauth.OAuthParameter(placement = BODY)
```

Comment:
maybe use the mule-extensions-api in one example to get more coverage?

Reply:
This feature will not be supported for those using extensions-api direclty, even if we can't enforce it